src/otx/cli/cli.py

```diff
@@ -331,11 +332,23 @@ def instantiate_classes(self, instantiate_engine: bool = True) -> None:
             # For num_classes update, Model and Metric are instantiated separately.
             model_config = self.config[self.subcommand].pop("model")
 
+            # if adaptive_input_size will be executed and the model has input_size_multiplier, pass it to OTXDataModule
+            if self.config[self.subcommand].data.adaptive_input_size != "none":
+                model_cls = get_model_cls_from_config(model_config)
+                self.config[self.subcommand].data.input_size_multiplier = model_cls.input_size_multiplier
```

Comment:
Is this necessary because different models have different `input_size_multiplier`?

Reply:
yes you're right. it depends on each model.